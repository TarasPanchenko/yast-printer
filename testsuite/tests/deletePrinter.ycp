/**
 * File:
 *  deletePrinter.ycp
 *
 * Module:
 *  Printer configurator
 *
 * Summary:
 *  Miscelaneous tests.
 *
 * Authors:
 *  Petr Blahos <pblahos@suse.cz>
 *
 * $Id$
 *
 */

{
    import "Pkg";

    include "testsuite.ycp";
    import "Printer";

    Printer::printcap = [
	$[ "name" : "1", ],
	$[ "name" : "2", ],
	$[ "name" : "3", ],
	$[ "name" : "4", ],
	];
    Printer::index = 0;
    Printer::deletePrinter ();
    DUMP (Printer::printcap);
    DUMP (Printer::DeletedNames ());

    Printer::printcap = [
	$[ "name" : "1", "saved" : $[ "printcap_part" : "yes"]],
	$[ "name" : "2", ],
	$[ "name" : "3", ],
	$[ "name" : "4", ],
	];
    Printer::index = 0;
    Printer::deletePrinter ();
    DUMP (Printer::printcap);
    DUMP (Printer::DeletedNames ());
}
