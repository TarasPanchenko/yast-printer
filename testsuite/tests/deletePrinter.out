Dump	[$["name":"2"], $["name":"3"], $["name":"4"]]
Dump	["1"]
Dump	[$["name":"2"], $["name":"3"], $["name":"4"]]
Dump	["1"]
